src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -78,26 +83,28 @@ public bool ExecuteCommand(FormatDocumentCommandArgs args, CommandExecutionConte
             }
 
             context.OperationContext.TakeOwnership();
+
             _waitIndicator.Wait(
                 EditorFeaturesResources.Formatting_document,
                 EditorFeaturesResources.Formatting_document,
                 allowCancel: true,
                 showProgress: true,
                 c =>
                 {
-                    var cancellationToken = context.OperationContext.UserCancellationToken;
+                    var docOptions = document.GetOptionsAsync(c.CancellationToken).WaitAndGetResult(c.CancellationToken);
```

Comment:
thanks for fixing this.  looks like i screwed this up when merging with @heejaechang 's changes.  we def do *not* want to use ```context.OperationContext.UserCancellationToken```.  Your code is correct and is important to get into 15.8.x.